### PR TITLE
added note about submitting multiple values using a comma, and added …

### PR DESCRIPTION
**Pull request by hknahal**
…examples

### CODE REVIEW

File: schemas/primary_diagnosis.json [hunk 1]
```diff
@@ -230,7 +230,11 @@
           "Weight Loss"
         ]
       },
-      "meta": { "displayName": "Presenting Symptoms" }
+      "meta": { 
+         "displayName": "Presenting Symptoms",
+         "notes": "To include multiple values, please separate values with a comma ',' within your file",
+        "examples": "Anemia,Bloating,Diabetes"
```
Comment (kcullion) [inline schemas/primary_diagnosis.json:236]: I may just say 
For example: Anemia, Bloting, Diabetes

As it's just one example

Comment (kcullion) [reply schemas/primary_diagnosis.json:236]: oh wait, never mind - is the word "examples" a dictionary variable thing

File: schemas/primary_diagnosis.json [hunk 2]
```diff
@@ -230,7 +230,11 @@
           "Weight Loss"
         ]
       },
-      "meta": { "displayName": "Presenting Symptoms" }
+      "meta": { 
+         "displayName": "Presenting Symptoms",
+         "notes": "To include multiple values, please separate values with a comma ',' within your file",
```
Comment (rosibaj) [inline schemas/primary_diagnosis.json:235]: period at end of sentence?

**Comment by hknahal** [comment]
Related to https://github.com/icgc-argo/argo-dictionary/issues/134

**Pull request merged**